payments/invoice/redirect.md

```diff
@@ -56,22 +75,342 @@ sequenceDiagram
     Consumer->>Merchant: Start purchase
     activate Merchant
     note left of Merchant: First API request
-    Merchant->>+PayEx: POST <Invoice Payment> (operation=FinancingConsumer)
-    PayEx-->>-Merchant: payment resource
+    Merchant->>+Swedbank Pay: POST <Invoice Payment> (operation=FinancingConsumer)
+    Swedbank Pay-->>-Merchant: payment resource
     Merchant-->>-Consumer: authorization page
-    note left of Consumer: redirect to PayEx
-    Consumer->>+PayEx: enter consumer details
-    PayEx-->>-Consumer: redirect to merchant
+    note left of Consumer: redirect to Swedbank Pay
+    Consumer->>+Swedbank Pay: enter consumer details
+    Swedbank Pay-->>-Consumer: redirect to merchant
     note left of Consumer: redirect back to Merchant
     Consumer->>Merchant: access merchant page
     activate Merchant
     note left of Merchant: Second API request
-    Merchant->>+PayEx: GET <Invoice payment>
-    PayEx-->>-Merchant: payment resource
+    Merchant->>+Swedbank Pay: GET <Invoice payment>
+    Swedbank Pay-->>-Merchant: payment resource
     Merchant-->>Consumer: display purchase result
     deactivate Merchant
 ```
 
+## API Requests
+
+The API requests are displayed in the [purchase flow](#purchase-flow).
+You can complete the invoice payment with following `operation`
+options:
+
+* [Financing Consumer][other-features-financing-consumer]
+* [Recur][recur]
+* [Verify][verify]
+
+Our `payment` example below uses the [`FinancingConsumer`]
+[other-features-financing-consumer] value.
+
+### Financing Consumer
+
+{:.code-header}
+**Request**
+
+```http
+HTTP/1.1 200 OK
+Content-Type: application/json
+
+{
+  "payment": {
+    "operation": "FinancingConsumer",
```

Comment:
Indent code examples with 4 spaces, please. The `prettyJson` extension can help with that.